README.md

```diff
@@ -152,6 +152,15 @@ cargo miri test
 When using the above instructions, you may encounter a number of confusing compiler
 errors.
 
+### "note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace"
+
+You may see this when trying to get Miri to display a backtrace. By default, Miri
+doesn't expose any environment to the program, so running
+`RUST_BACKTRACE=1 cargo miri test` will not do what you expect.
+
+To get a backtrace, you need to disable isolation
+[using `-Zmiri-disable-isolation`](#miri-flags)
```

Comment:
Could there be a flag to pretend that `RUST_BACKTRACE=1` is set even when isolation is enabled?

Reply:
We could - do you think that to opt-out of isolation only for `RUST_BACKTRACE` is a common usecase?

I think it might be better to add more fine-grained isolation in a separate PR. We could add `-Zmiri-env-include=VAR`, which would pass through `VAR` to the emulated program while leaving the isolation mode unaffected.

Reply:
Both of these seem like potentially reasonable next steps, but the PR is useful even without them.